### PR TITLE
ProxyNetworkConfigTest test create

### PR DESCRIPTION
**Pull request by Meylin-V**

### CODE REVIEW

File: src/test/java/executor/service/model/ProxyNetworkConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    @Test
+    public void equalsTest() {
+        ProxyNetworkConfig pNC = new ProxyNetworkConfig(HOSTNAME, PORT);
+        ProxyNetworkConfig pNCAnother = new ProxyNetworkConfig(NEW_HOSTNAME, NEW_PORT);
+
+        assertEquals(proxyNetworkConfig, pNC);
+        assertNotEquals(proxyNetworkConfig, pNCAnother);
+        assertNotEquals(proxyNetworkConfig, null);
+    }
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyNetworkConfigTest.java:61]: Let's add a tests to check 'equals' and 'hashCode' for empty object comparing with null and existing ProxyNetworkConfig.
First of all you need to add a default constructor to ProxyNetworkConfig to have a chance do this checks.

**Comment by Meylin-V** [comment]
test was added, ready to review

**Review by maksymivanov2510 — APPROVED** [review]
Looks good! Let's merge the branch
